DBSCE/Overview.md

```diff
@@ -0,0 +1,221 @@
+  - [Authors](#authors)
+  - [Contributors](#contributors)
+  - [Participate (TBD links)](#participate-tbd-links)
+  - [Overview](#overview)
+  - [Why DBSC(E)?](#why-dbsce)
+  - [How does it integrate with DBSC?](#how-does-it-integrate-with-dbsc)
+  - [High-Level Design](#high-level-design)
+  - [Terminology](#terminology)
+    - [Browser](#browser)
+    - [Relying Party (RP)](#relying-party-rp)
+    - [Identity Provider (IdP)](#identity-provider-idp)
+    - [Device Registration Client](#device-registration-client)
+    - [Local Key Helper](#local-key-helper)
+      - [Platform Examples:](#platform-examples)
+    - [Attestation Service:](#attestation-service)
+    - [Device Registration (Pre-Session)](#device-registration-pre-session)
+    - [DBSC(E) use cases](#dbsce-use-cases)
+      - [IDP Calls Public Local Key Helper](#idp-calls-public-local-key-helper)
+      - [IDP is RP and Calls Public Local Key Helper](#idp-is-rp-and-calls-public-local-key-helper)
+      - [IDP Calls Private Local Key Helper](#idp-calls-private-local-key-helper)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Device Bound Session Credentials for Enterprise - explainer
+
+This is the repository for Device Bound Session Credentials for Enterprise. You're welcome to
+[contribute](CONTRIBUTING.md)!
+
+## Authors
+
+- [Sameera Gajjarapu](sameera.gajjarapu@microsoft.com), Microsoft
+- [Aleksander Tokarev](alextok@microsoft.com), Microsoft
+
+## Contributors
+
+- [Olga Dalton](), Microsoft
+- [Kristian Monsen](), Google
+- [Phil Leblanc](), Google
+- [Sebastian](), Google
+- [Arnar Birgisson](), Google
+- [Pamela Dingle](), Microsoft
+- [Paul Garner](), Microsoft
+- [Erik Anderson](), Microsoft
+- [Will Bartlett](), Microsoft
+- [Kai Song](), Microsoft
+- [Amit Gusain](), Microsoft
+
+## Participate (TBD links)
+
+- [Issue tracker]()
+- [Discussion forum]
+
+## Overview
+
+Device Bound Session Credentials for Enterprise - DBSC(E), is an enhancement to the existing [DBSC](https://github.com/wicg/dbsc) proposal. It refines the key generation mechanism resulting in additional security for enterprise use cases. It aims to provide a mechanism for enterprise and advanced browser customers to be able to deploy enhanced/customised device binding for any browser session, hence protecting against session hijacking and cookie theft.
+
+## Why DBSC(E)?
+
+While the original DBSC proposal is focused on providing a mechanism for browsers to bind session credentials to a device, it still remains vulnerable to malware that can run on a device during any web application signin/login. If a malware happens to be already running in the device, it can force the user to login, and provide its own binding keys (asymmetric key pair) to the web application, there by gaining the ability to steal the session. Any upcoming sessions after this, even with DBSC, will not be reliable.
```

Comment:
I think one thing that I'd like to highlight a bit more here is the difference between persistent vs temporary malware. With the original DBSC, even temporary infection can result in a permanent session compromise. With DBSC(E), malware would have to persist on the device and exfiltrate data rather than sessions. 

I took a stab at changing a couple of sentences to highlight this difference a bit more:

```
While the original DBSC proposal enables browsers to bind session cookies to a device, it doesn’t protect against on device malware. Such a malware can inject binding keys that it controls when DBSC session gets established during user’s login operation. It can also force user to login again if needed and steal the session at that time. This means that even a temporary malware infection can result in permanent session compromise. 


DBSC(E) aims to mitigate this risk by introducing the concept of one time device registration and binds all the future sessions to a binding key that is cryptographically proven to be on the same device. Because device registration is expected to be a once-in-a-lifetime protected operation, the user will not be required to perform this operation again, reducing opportunities for malware to permanently extract session credentials of the device.  Device registration must happen when there’s no malware on the device (a state referred to as “clean room”), e.g. before giving a device to an employee. If this condition is satisfied, DBSC(E) makes it impossible for malware to permanently compromise user’s session during sign in operation. However, DBSC(E) doesn't protect if the malware is present during the device registration, or if malware is persistent on the device and uses device-bound sessions to exfiltrate user’s data. 
```

Reply:
Updated with slight changes.

Reply:
Added more precise language for the mitigation, attack details and the short term risk associated. Let us track this with a different comment as we get more feedback. Closing this comment.